src/downloaders.py

```diff
@@ -0,0 +1,52 @@
+"""
+Downloaders for docs stylesheet, parser and stdlib.
+"""
+import constants
+from download_helpers import download_from_url, download_from_git
+from replace_all_occurences_in_file import replace_all_occurrences_in_file
+
+
+def download_stylesheet() -> None:
+    """
+    Downloads the stylesheet from the IDE's repository.
+    """
+    style_directory: str = (
+        constants.IDE_BRANCH + "/src/rust/ide/view/src/documentation/"
```

Comment:
This should also be a constant. Anything that can change externally should be a constant.